sdk/communication/communication-call-automation/recordings/CallConnection_Live_Tests_Add_a_participant_and_get_call_properties.txt

```diff
@@ -0,0 +1,16 @@
M2h1U1dwdmFWVnNUWGxPVkZscFRFTktjbUZYVVdsUGFVcFZZbGhLVjFGdWJESmtNa2w1WkZVNVYxUXhUakZPTTJ3d1RWVjBTMUpYV25SWk1UbFRWa2RTU2s1WFNsQlBSbFpKV1RKamQxVnFiRE5KYmpFNVRFTktjRnBJUVdsUGFVcHZaRWhTZDJONmIzWk1NMDR3WTNrMU0yRlhOV3RpTTJSNlRHMDFiR1JET1dwYVIwMHhXVmRXYkZsVE1IaE9WMDB4VEZSU2ExbHFXWFJaYWtFelQxTXhiVmt5Um10YVJFa3hUVVJXYTFsNlNYWkphWGRwWWpKc2EwbHFiMmxOYlUwelQxZE9hVTU2WjNSTk1rNXFXVk13TUUweVZtbE1WMHBzV1dwSmRGbHFRVFZPYlZacFQxUkpNRTV0UlRSSmFYZHBZMjFuYVU5cFNYZE1hMFpUVTFWRk1tTlVaRWRsYlU1V1ZtNVNjazF1Wkd4YWJtd3dUVWQ0UTFwSVpHNWlNSEJaWTJ4T1prMXRkRmRSYlRsSVdqQnNlRkV5TVRaU1J6aDNWVEJHUWxGVE5HbE1RMHA2WkZkSmFVOXBTWGxaZW1NMVdUSkpNMDlETUhwWk1rNW9URlJSZWxwWFNYUlpiVlpwVFdreGFVMUVhekphVjBrMVRXcFJNbGxVWjJsTVEwb3dZVmRSYVU5cFNtcGFSMDB4V1ZkV2JGbFRNSGhPVjAweFRGUlNhMWxxV1hSWmFrRXpUMU14YlZreVJtdGFSRWt4VFVSV2ExbDZTV2xNUTBveFpFZHJhVTlwU2xwa2JXeG1WVzFrU0dSVlRYZFdNM0JNVlROR2JWVllTa2RaTUVaQ1NXbDNhV1J0Vm5sSmFtOXBUVk0wZDBsdU1DNTRhMlI1VXpkWVFWcGtTVUZQYlRFMmVGWmFiRlZyTXpCalZGSnJPSE5LVkhWYVYwOVRjVzlXWkhWWlJrRlpNM0ZTVG5kc05HcG9OSGRZY2tWME1FcEtkSFpvZDIxWlQyWlJRMTlvV0hoSFNpMU9WVjlJUWxsVE9GQlFVRE5YUjJod1EwUkdSbGs0YjE5blIxZ3pOUzFZWkVWTFF6aElTamhrWm5KVlREQmhXRkpOVWxnelNFeDNVMGxmTld4MVkwcDNWekY1ZFRCWk9WQnBXbFJUTW5rNFkxUkdhV3BOYjFwc1MyaE5RbEpUUTNCdGFGVXdiVGxFUmtoWGRFRmZaRFpxZDJsYU4zVllOVWxPVHpoRExYYzJOVGh2UkMxemRGaHBZemM1VldadmNHRlhjMFUyUW5ZNE5sTTJWRmhVUTNsQlJHZHpMVVl5UzJkcWFsQnpaVmQ1T1VwZlFqTkdTbGRIYURGSlZFbERNRzFMV0RkUk5YZE1VM295VkhsYVZGcG9NalZJWVhCcUxURldNRkp2UW5kMVUwMDNNR0ZzZVRORlVYVmpXVW95ZVd0V1FsQlNiMEpuUlZkQk4wcDBkMEpIWm1vMFVuY2lMQ0owY3lJNk1UY3hNelE0TVRJMk1Td2lZaUk2SWpJMVJtOURPR2xpTjJWM1NXTjZkbVZ0TVZoSlgxOVpTWGgwWms5ME5sQllZVFpuVm5VMVYySk9VazBpTENKdWIyNWpaU0k2SW1ReFl6UmpNV0kzTldFME16UXpaalE0TjJZek9HUTVZalpsT1RBeFlUSTVJaXdpWTI1bUlqcDdJbXAzYXlJNmV5SnJhV1FpT2lKR09EQXdWSFZFY1dod1RHOUpibEkyY1Rac1oxYzBMVXRFZVV0Q2RWb3dXRjl2VUZOeU1rRnFXbFV3SWl3aVpTSTZJa0ZSUVVJaUxDSnJkSGtpT2lKU1UwRWlMQ0p1SWpvaWRGQm1Xa1ZSTWxsRlgwdGFPRlk1YjJ0S2VtMDVUWE5hVm1SUlpGSnJRblJyWlVKZk0yNXpURFp5UkdoM2NEQXlZbXhsWkdsbFFta3hSbEpVVDAxNGFWaEpja1ZZUTFWNFZDMDNObGhFTTAxUFptMDFkVGQ0WmpobFFVeENWVkV4TkhGMU1GbFJUMmRLUVhadVVGWlhkRmhCZUZoMFl6bFBWSE5aY1ZZMWNYTmpSRVpRZEd0NVMzYzRNRW8zY2pWTVdFWkpOMnM0Y25kNVYwaEpURXMxZFhnemFYWmxhakpQVEVOdlEzVjJkakZZVEc5WWVtVnZhVWhpUXpObU16TnZORXg2TURWZmRFOUhSMnBSYmpCUE5sVlFUblZPVlV0VVpuTkRVMFF4WkVkdVQwWlFRekYyYmsxZlluRmxWalpuZDA5eGFFSmZaVjlrZFRGUloyNXJRWEJyZVVOVFVFSndiRFJyYkhwYVR6bHZVVXR6TFZGbGNXSk1aMVJCWW5SMVUyeEhjVFptYmpCQ2RtRk9XRWN5ZUhaalZrdDVZMk5rUmxGRU5rVnJNWGhrVDBseWNVRkZkRWx6ZDBoTVEwczFkVmhXT0Y5UkluMTlmUS5JelZoRWo5b2kwaV8xU2UxYlVnd2RVVnVsVER4dkZ6dGJaNHdMOWdwLWZDNGJ3NEVWSXE2cUhLQ0JxX3paNERCWDc0bWM3dHdjcHZ2Yk1ucDJZZ1hhc0REbkdqMzByOGxsNWRIZTZXMDJrOXU2V2VfWjdINUZBR3Z2QmtRS3BCSjc0Q1A2b3FpYzRldGwybVBTOEpRMzZqSTh1aExEbTRraEFFSGpDcmQ1TGNGM1VmT0ZwVDU1aW44WldJR1lEdHpUekoteEdhZDYtQnpWOWRNSU1DRHBBeFdJejdhOXRITlZ2emNxVFdGa3JCM2pnek1KdVlCbE5rTXk0NUVaY19Db3JsLU5EX0MwX2FMMEVCaE1LZmFEa0dFZ2dTa2xVZF9ZeVV0YzVLS2hydVRFcTJPQzJmeUgtOG5tckRqNXB1c0R1QWlQNTJYMWUtTHVWbng1TXVJU2cifQ.","correlationId":"2f03a068-fc6a-40a2-ba14-9ce5594849b5"}
```

Comment:
@harsha-nalluru do you know why the recording files are txt and back to sdk repo?

Reply:
[Golden Testing Commands · Azure/azure-sdk-for-js Wiki (github.com)](https://github.com/Azure/azure-sdk-for-js/wiki/Golden-Testing-Commands)

`npx dev-tool test-proxy init`
`npx dev-tool test-proxy push`

Reply:
I'll pull the branch and see why they are not in the assets repo

Reply:
Oh I just saw the files are txt. I am guessing it may be a change from @scbedd
I'll wait till here replies.

Reply:
@HarshaNalluru the recorder api is providing a path ending with `.txt` and not `.json` would be my guess.